proxy/hdrs/HdrTest.cc

```diff
@@ -522,6 +524,16 @@ HdrTest::test_mime()
     return (failures_to_status("test_mime", 1));
   }
 
+  // Test the (new) continuation line folding to be correct. This should replace the
+  // \r\n with two spaces (so a total of three between "part1" and "part2").
+  int length;
+  const char *continuation = hdr.value_get("continuation", 12, &length);
+
+  if ((13 != length) || strncmp(continuation, "part1   part2", 13) || strncmp(continuation + 5, "   ", 3)) {
```

Comment:
This will be easier to debug if you separate each condition into a separate failure.